resources/js/actions/userAuthActions.js

```diff
@@ -0,0 +1,77 @@
+import userApi from '../apis/userApi';
+import { SIGN_IN } from './types';
+
+export const signIn = (formData) => async dispatch => {
+    var data = {};
+    var submitDisabled = true;
+    const email = formData.get('email');
+    const password = formData.get('password');
+    const users = {
+        email: email,
+        password: password
+    }
+    // if all fields are not blank 
+    if (users.email != '' && users.password != '') {
+        await userApi.post('/sign-in', users)
+        .then(response => {
+            // this is used for error checking display
+            data = {
+                submitDisabled: false,
+                errors: response.data.errors,
+                message: response.data.message
+            }
+        })
+    }else if (users.email == '' && users.password != '') {
+        data = { submitDisabled ,email: 'Email should not be empty' }
+    }else if (users.email != ''  && users.password == '') {
```

Comment:
same here apply strict inequality opetator, and put space after `}`

```suggestion
    } else if (users.email !== ''  && users.password === '') {
```

Reply:
Made changes to userAuthActons. This is resolved.